homeassistant/components/recorder/const.py

```diff
@@ -19,7 +19,7 @@
 
 CONF_DB_INTEGRITY_CHECK = "db_integrity_check"
 
-MAX_QUEUE_BACKLOG = 40000
+MAX_QUEUE_BACKLOG = 65000
```

Comment:
The tests need to be adjusted

Reply:
fixed in [110fcc5](https://github.com/home-assistant/core/pull/87090/commits/110fcc55e7e10ccd75de558ebfec55d6605758e2)